src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/NuGetProjectManagerService.cs

```diff
@@ -398,6 +399,19 @@ public async ValueTask ExecuteActionsAsync(IReadOnlyList<ProjectAction> actions,
             DependencyBehavior dependencyBehavior,
             IReadOnlyList<string> packageSourceNames,
             CancellationToken cancellationToken)
+        {
+            return await GetInstallActionsAsync(projectIds, packageIdentity, versionConstraints, includePrelease, dependencyBehavior, packageSourceNames, cancellationToken, versionRange: null);
+        }
+
+        public async ValueTask<IReadOnlyList<ProjectAction>> GetInstallActionsAsync(
+            IReadOnlyCollection<string> projectIds,
+            PackageIdentity packageIdentity,
+            VersionConstraints versionConstraints,
+            bool includePrelease,
+            DependencyBehavior dependencyBehavior,
+            IReadOnlyList<string> packageSourceNames,
+            CancellationToken cancellationToken,
+            VersionRange? versionRange)
```

Comment:
Convention is that CancellationToken is always the last parameter. I don't know why, it just is 😕